var/spack/repos/builtin/packages/apptainer/package.py

```diff
@@ -33,6 +33,7 @@ class Apptainer(SingularityBase):
     )
 
     version("main", branch="main")
+    version("1.3.1", sha256="6956c689c4a8f148789c5c34b33c15ad8f3460b4cee3f48022119fd872eacee9")
```

Comment:
Due to changes in behavior between apptainer versions, does it make sense to add a 1.2 as well?
```suggestion
    version("1.3.1", sha256="6956c689c4a8f148789c5c34b33c15ad8f3460b4cee3f48022119fd872eacee9")
    version("1.2.5", sha256="606b67ef97683e1420401718687d258b1034fdf2edae72eeacd0828dffbfc2c2")
```